src/service.rs

```diff
@@ -18,7 +18,7 @@ use std::cell::OnceCell;
 use std::rc::Rc;
 use std::time::Duration;
 
-#[derive(Default, Debug)]
+#[derive(Default, Clone, Debug)]
```

Comment:
Why is this cloned now? I see we store `Rc<GrpcService>` in out "actions" 🤔

Reply:
I do not think it is needed. It *was* for something I tried. Not anymore.